core/dbt/clients/yaml_helper.py

```diff
@@ -20,6 +21,19 @@
 """.strip()
 
 
+class UniqueKeyLoader(SafeLoader):
```

Comment:
Can you link to where you found this solution?  It gives some great contect as to why we're doing this which will be 100% helpful down the road when we ask "but why".